### PR TITLE
[v6r19] fixes to IS resolution

### PR DESCRIPTION
**Pull request by fstagni**
For #3370

### CODE REVIEW

**Comment by andresailer** [comment]
This doesn't work for me

**Comment by andresailer** [comment]
It fails if there are files in the sandbox that are in the working directory when the job submission happens. My PR, however works.

File: Interfaces/API/Dirac.py [hunk 1]
```diff
@@ -859,32 +859,33 @@ def runLocal(self, job):
...
-        elif os.path.exists(os.path.join(jobDir, isFile)): # if it is in the tmp dir
-          self.log.debug("Input Sandbox %s is a file, found in the job directory, no need to copy it" % isFile)
-        else:
-          self.log.verbose("perhaps the file %s is in an LFN, so we attempt to download it." % isFile)
-          getFile = self.getFile( isFile )
-          if not getFile['OK']:
-            self.log.warn( 'Failed to download %s with error: %s' % ( isFile, getFile['Message'] ) )
-            return S_ERROR( 'Can not copy InputSandbox file %s' % isFile )
-        basefname = os.path.basename( isFile )
+        else: # Attempt to find it and copy into job working directory
```
Comment (andresailer) [inline Interfaces/API/Dirac.py]: Is this else with all the indentation changes necessary? Can't you just change the first `if` to `elif` for the same behaviour?
That would make the diff easier to parse

**Comment by andresailer** [comment]
I have already rebased to v6r19 and changed the target branch. I am fairly sure there will be conflicts

**Comment by fstagni** [comment]
@atsareg please merge first @andresailer PR and then if there will be conflicts with this one I will sort them out.

**Comment by atsareg** [comment]
Conflicts as expected. Please, fix

**Comment by fstagni** [comment]
Conflicts fixed.

**Pull request merged**